### ISSUE TITLE
Can https://redhat-documentation.github.io/asciidoc-markup-conventions/ be incorporated into the SSG?

### ISSUE THREAD

**Issue by max-cx** [redhat-documentation/supplementary-style-guide#233]
There are markup conventions in https://redhat-documentation.github.io/supplementary-style-guide/#formatting.

There are markup conventions in https://redhat-documentation.github.io/asciidoc-markup-conventions/.

Can all these conventions somehow reside in one place?

**Comment by bburt-rh** [comment]
Thanks for bringing this up, @max-cx. My own view is not that they should be in one place but that the SSG should be "platform agnostic" and NOT contain coding guidance for a specific markup language -- i.e., that coding guidance should be maintained separately. My thinking is that that SSG guidance should not be married to a specific markup language because RH or specific docs projects within RH could use languages different than AsciiDoc.

So I think it's worth discussing whether or not we want to move markup conventions out of the SSG and into the Markup Conventions doc rather than merging everything into one doc.

@bergerhoffer WDYT? You have more historical knowledge of why AsciiDoc markup conventions might be included in the SSG. What's the thinking? Is there a reason why this info should be in the SSG? Thanks!

**Comment by bergerhoffer** [comment]
Yeah the general consensus at the creation of the SSG is that we did want to try to be markup-agnostic and not get into specifics of whether you should do it this way in AsciiDoc or that way in AsciiDoc.

That being said, AsciiDoc is what we use, and we did see the value in providing some examples where it would be helpful. If we were to change to another language, it wouldn't be hard to update these examples as needed.

We even address this with this statement here: https://redhat-documentation.github.io/supplementary-style-guide/#style-guidelines:

> If applicable, these guidelines provide example formatting in AsciiDoc, which is the markup language that Red Hat Customer Content Services currently uses. However, the guidelines can be applied to any language.

I personally think that this is the right balance. I'm not in favor of combining the AsciiDoc markup guide into the SSG, because that's not what we're trying to be. Truly the markup guide could easily get enhanced, but it has pretty much stayed the same since we created that years ago (there isn't really a project team behind it anymore - one or two of us are still around and might check if something popped up).

But I'm also not in favor of removing the AsciiDoc examples that we have. I think we've done a good job of not overusing them. 

For example, https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values gives some examples of using the replaceable value in AsciiDoc. I think this is helpful if someone didn't know exactly how to do the syntax when it's in a code block (you have to use "subs"). 

We do also reference the AsciiDoc guide from the SSG here: https://redhat-documentation.github.io/supplementary-style-guide/#_related_red_hat_guides

We also reference the mod docs guide. IMO it's the same kind of thing - the true guidelines for mod docs lives in the mod docs guide, but that doesn't mean that the SSG doesn't ever talk about mod docs rules.

So long story short, from me, I'd prefer to keep things as they are now. :slightly_smiling_face:

**Comment by bburt-rh** [comment]
@bergerhoffer Thank you for that detailed and nuanced reply. I understand the reasoning, and it makes sense to me. 

What it comes down to in the end for me is that the priority should always be to make the guides as useful as possible for our audience, and the way you describe the intent of the guide seems to give us some flexibility in this regard to include markup-related content when we think it will be useful. A strict policy separating out the two content types wouldn't provide this flexibility. 

So, in sum, you've convinced me, and I'm on board with keeping things as-is, as long as it's clear to everyone that the SSG is not meant to be a comprehensive AsciiDoc reference. Maybe we should consider putting in a link to the AsciiDoc reference guide, though, whenever we provide AsciiDoc guidance in the SSG? 

I would be interested to know what @max-cx and others think regarding this issue, though.

**Comment by bergerhoffer** [comment]
Discussed this at the 30 November meeting.

Consensus was in favor to keep these two guides separate. The SSG does not attempt to be an AsciiDoc guide, though it does occasionally give examples in AsciiDoc to help readers. If we were to change languages in CCS, we would change those examples. We prefer to keep our style guide mainly about style, and to leave AsciiDoc specifics up to that guide. The SSG makes sure not to conflict with any guidance in the AsciiDoc writing guide.

Based on this, I am closing this issue. Thanks!
